tests/python_client/testcases/test_collection.py

```diff
@@ -2725,3 +2710,117 @@ def test_load_release_after_collection_drop(self, connect, collection):
+        self.collection_wrap.init_collection(name=c_name, schema=schema, check_task=CheckTasks.check_collection_property,
+                                             check_items={exp_name: c_name, exp_schema: schema})
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_collection_only_string_field(self):
+        """
+        target: test create collection with one string field
+        method: create  collection with only string field
+        expected: Raise exception
+        """
+        self._connect()
+        string_field = cf.gen_string_field(is_primary=True)
+        schema = cf.gen_collection_schema([string_field])
+        error = {ct.err_code: 0, ct.err_msg: "No vector field is found"}
+        self.collection_wrap.init_collection(name=cf.gen_unique_str(prefix), schema=schema,
+                                             check_task=CheckTasks.err_res, check_items=error)
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_collection_string_field_with_exceed_max_len(self):
+        """
+        target: test create collection with string field
+        method: 1. create  collection with string field
+                2. String field max_length_per_row exceeds maximum
+        expected: Raise exception
+        """
+        self._connect()
+        c_name = cf.gen_unique_str(prefix)
+        int_field = cf.gen_int64_field(is_primary=True)
+        vec_field = cf.gen_float_vec_field()
+        max_length = 100000
+        string_field = cf.gen_string_field(max_length_per_row=max_length)
+        schema = cf.gen_collection_schema([int_field, string_field, vec_field])
+        error = {ct.err_code: 0, ct.err_msg: "invalid max_length_per_row: %s" %max_length}
+        self.collection_wrap.init_collection(name=c_name, schema=schema, 
+                                             check_task=CheckTasks.err_res, check_items=error)
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_collection_invalid_string_field_dtype(self):
+        """
+        target: test create collection with string field
+        method: create collection with string field, the string field datatype is invaild
+        expected: Raise exception
+        """
+        self._connect()
+        string_field = self.field_schema_wrap.init_field_schema(name="string", dtype=DataType.STRING)[0]
+        int_field = cf.gen_int64_field(is_primary=True)
+        vec_field = cf.gen_float_vec_field()
+        schema = cf.gen_collection_schema(fields=[int_field, string_field, vec_field])
+        error = {ct.err_code: 0, ct.err_msg: "string data type not supported yet, please use VarChar type instead"}
+        self.collection_wrap.init_collection(name=cf.gen_unique_str(prefix), schema=schema,
+                                             check_task=CheckTasks.err_res, check_items=error)
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_collection_invalid_string_field_with_auto_id(self):
```

Comment:
The test case name did not reflect the functions tested, and the  "test_collection_string_field_is_primary_and_auto_id"  may be better. and the following comment should be modified to reflect what is tested.

Reply:
Thanks for the suggestion has been modified